src/components/date-input/template.test.js

```diff
@@ -74,6 +74,40 @@ describe('Date input', () => {
 
       expect($items.length).toEqual(3)
       expect($firstItemInput.attr('name')).toEqual('day')
+      expect($firstItemInput.attr('pattern')).toEqual('[0-9]*')
```

Comment:
Non-blocking, but this feels like it might be important enough to be split out into its own test (I think we're also trying to have one expectation per test, but we're not super strict on that)

Reply:
That seems to be tested already in "renders inputs with pattern="[0-9]*" to trigger numeric keypad on iOS", so we could consider removing this assertion here.

Reply:
Good spot @nickcolley! 👍Looks like this is duplicated in that case?

Reply:
No problem, now removed! Thanks both